src/zjs_dgram.c

```diff
@@ -295,10 +293,13 @@ static jerry_value_t zjs_dgram_sock_bind(const jerry_value_t function_obj,
                                          const jerry_value_t argv[],
                                          const jerry_length_t argc)
 {
-    int ret;
+    // NOTE: really argv[0] and argv[1] are supposed to be optional, but
+    //   leaving them as they were here for now
```

Comment:
Current googleable Node.js docs show them as optional, but Node.js I have on my Ubuntu 14.04 requires them and doesn't allow to omit. And then googleable docs don't say anything from which version of Node.js they became optional. In such situation, I consider a real-world executable to be definitive, whereas randomly googled-up docs can be for too old a version (was optional, but no longer), or too new (will be become optional in a version which will get world-wide deployment in a year). Generally, I find Node.js docs lacking many important details, that's why I asked you guys if you have a habit of poking into Node.js source when unclear cases arise (it's in my TODO to acquire such a habit ;-) ).